Gemfile.lock

```diff
@@ -498,3 +500,6 @@ DEPENDENCIES
   xapian-full-alaveteli (~> 1.2.21.1)
   xml-simple (~> 1.1.0)
   zip (~> 2.0.0)
+
+BUNDLED WITH
```

Comment:
shame we can't configure bundler not to do that

Reply:
Sorry, I keep missing that.